app/jobs/job_posting_poll_job.rb

```diff
@@ -4,6 +4,6 @@ class JobPostingPollJob < ApplicationJob
   queue_as :default
 
   def perform(*)
-    JobPostingPollerService.poll
+    JobPostingApi::Poller.new.perform
```

Comment:
Don't you wan't perform_later? And then sidekiq starts as soon as it's ready? Maybe in the future we have many more ActiveJobs?

Reply:
It's calling the service. The job itself you still can perform_later

Reply:
Aaah, you named the "execute" function of your Service perform as well. That confused me.